adal/src/Microsoft.IdentityModel.Clients.ActiveDirectory/Platforms/netstandard1.3/PlatformInformation.cs

```diff
@@ -25,7 +25,6 @@
 //
 //------------------------------------------------------------------------------
 
-using Microsoft.IdentityModel.Clients.ActiveDirectory.Internal.Platform;
 using System.Threading.Tasks;
```

Comment:
is this needed?

Reply:
It was greyed out and VS said it was unnecessary, so I removed it.